lib/public/Calendar/ICalendar.php

```diff
@@ -39,6 +39,11 @@ interface ICalendar {
 	 */
 	public function getKey();
 
+	/**
+	 * @since 24.0.0
+	 */
+	public function getUri();
```

Comment:
As this is a new function, please strong type.
```suggestion
	public function getUri(): string;
```